File: Sem5Task34/Program.py
#34
import random
list = [] #создаем пустой список
count = 0  
num = int(input())  #вводим число
for i in range(num):  
    rnd_list = random.randrange(100, 999)  #создаем числа 
    list.append(rnd_list)  #и вставляем их в список
for i in range(len(list)):   #создаем новый цикл на длину массива 
    if list[i]%2==0:   #если число четное
        count+=1    #вставляем в переменную каунт
print('Массив:',list,'Колличество четных элементов', count)
#34*
def bubble(list):
    n = len(list)
    for i in range(n - 1):
        for j in range(n - i - 1):
            if list[j] > list[j + 1]:
                list[j], list[j + 1] = list[j + 1], list[j]
    return list
print(bubble(list))

File: Sem5Task34/Program.cs
int ReadData(string msg)
{
    Console.WriteLine(msg);
    return int.Parse(Console.ReadLine() ?? "0");
}

void PrintData(string res)
{
    Console.WriteLine(res);
}

int[] Gen1DArr(int num, int begin, int end)
{
    Random rnd = new Random();
    int[] arr = new int[num];
    for (int i = 0; i < arr.Length; i++)
    {
        arr[i] = rnd.Next(begin, end);
    }
    return arr;
}

void Print1DArr(int[] arr)
{
    Console.Write("[");
    for (int i = 0; i < arr.Length - 1; i++)
    {
        Console.Write(arr[i] + ", ");
    }
    Console.WriteLine(arr[arr.Length - 1] + "]");
}

int CountElem(int[] arr)
{
    int res = 0;
    for (int i = 0; i < arr.Length; i++)
    {
        if (Test(arr[i]))
        {
            res++;
        }
    }
    return res;
}

bool Test(int n)
{
    return n%2==0;
}
int arrLen = ReadData("Введите длину массива.");
int[] testArr = Gen1DArr(arrLen,100,1000);
Print1DArr(testArr);
int count = CountElem(testArr);
PrintData("Количество четных элементов в отрезке [100,999] :"+count);